cdpr:
   # Settings for ros_control hardware interface
   hardware_interface:
      loop_hz: 100 # hz
      joints:
        - joint_steer_lf
        - joint_steer_rf
        - joint_steer_lb
        - joint_steer_rb
        - joint_roll_lf
        - joint_roll_rf
        - joint_roll_lb
        - joint_roll_rb
      joint_mode: 1  # 1: position     2: velocity    3: effort
